### PR TITLE
Quickstart 2 - Docs Updated for V6

### PR DESCRIPTION
**Pull request by josephdecock**

### CODE REVIEW

File: IdentityServer/v6/docs/content/quickstarts/1_client_credentials.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ You can now use your favorite text editor to edit or view the files.
 Add the IdentityServer project to the solution:
 ```console
 cd ..
-dotnet sln add ./src/IdentityServer/IdentityServer.csproj
+dotnet sln add .\src\IdentityServer\IdentityServer.csproj
```
Comment (leastprivilege) [inline IdentityServer/v6/docs/content/quickstarts/1_client_credentials.md:67]: IIRC forwards slashes work xplat. Backwards not

Comment (josephdecock) [reply IdentityServer/v6/docs/content/quickstarts/1_client_credentials.md:67]: Oh shoot, that's right. Fixed!

**Comment by brockallen** [comment]
Hey -- did the last set of changes get picked up in the build/publish? I don't see them.

Also, in general, keep in mind the cross-plat audience, so we want OS/shell things that work in both and/or are obvious what someone would need to do on their platform. So slashes in paths, shell commands, etc. TY!

**Comment by brockallen** [comment]
> Hey -- did the last set of changes get picked up in the build/publish? I don't see them.

Oh, this might be because the staging site only can handle on PR at a time? @leastprivilege do you recall how it works?

File: IdentityServer/v6/docs/content/quickstarts/2_interactive.md [hunk 2]
```diff
@@ -243,30 +166,185 @@ public static IEnumerable<Client> Clients =>
...
 
-Trigger the authentication handshake by navigating to the protected controller action.
-You should see a redirect to the login page of the IdentityServer.
+## Test the client
+Now everything should be in place to log in to *MvcClient* using OIDC. Run
+*IdentityServer* and *MvcClient* and then trigger the authentication handshake by
+navigating to the protected controller action. You should see a redirect to the
+login page in *IdentityServer*.
 
-![](../images/3_login.png)
```
Comment (leastprivilege) [inline IdentityServer/v6/docs/content/quickstarts/2_interactive.md:252]: I think we should update the MVC client screenshots. Some claims are gone, some have been added (it also says "copyright 2016" ;))

File: IdentityServer/v6/docs/content/quickstarts/2_interactive.md [hunk 3]
```diff
@@ -291,90 +388,160 @@ You might have noticed that even though we've configured the client to be allowe
...
+IdentityServer can be added as an additional external provider.
+
+Register and configure the services for the OpenId Connect handler in
+*IdentityServer/HostingExtensions.cs*:
+```cs
+builder.Services.AddAuthentication()
+    .AddGoogle("Google", options => { /* ... */ })
     .AddOpenIdConnect("oidc", "Demo IdentityServer", options =>
     {
         options.SignInScheme = IdentityServerConstants.ExternalCookieAuthenticationScheme;
```
Comment (leastprivilege) [inline IdentityServer/v6/docs/content/quickstarts/2_interactive.md:547]: please add

options.ResponseMode = "query";

**Pull request merged**